src/app/components/icons/IconBrowser.tsx

```diff
@@ -150,7 +150,7 @@ const loadIcons = (): void => {
 
         const icon: IconType = {
             name: iconKey,
-            iconFontKey: iconDetails.name,
```

Comment:
Was there a reason for changing the key used for the BLUI icons?  Fine with the change regardless but wondering if iconDetails.name isn't guaranteed as unique.

Reply:
I think the intent here is, icon.name is the "MUI name", while iconFontKey would be the filename without the ".svg" part. So if I have an icon called "px-blue-icon.svg", the `icon.name` field would be `PxBlueIcon`, and the `icon.iconFontKey` should store `px-blue-icon`.

`iconDetails.name` stores the filename in a human-readable way, in Title Case. So using the example above,  `iconDetails.name` would be `PX Blue Icon`. However, this thing has some legacy issues with it and can cause some confusion, so it is not actively used anywhere.